README.md

```diff
@@ -163,8 +163,19 @@ go get k8s.io/kube-state-metrics
 ```
 
 #### Building the Docker container
+Prior to making the container you may want to use godeps the restore and save 
+the dependencies.  You may also be missing oauth2 from your current environment.  
+You can perform the godeps save first to see if you are missing oauth2.  
```

Comment:
This is actually a developer guide instead of the preliminary for `make container`. Also, by using golang vendor, no else dependencies of kube-state-metrics should be `go get`, i.e., all dependencies of kube-state-metrics are all under vendor directory.